test/unit/uk/gov/hmrc/apinotificationpull/controllers/NotificationsControllerSpec.scala

```diff
@@ -98,25 +101,21 @@ class NotificationsControllerSpec extends UnitSpec with WithFakeApplication with
 
       protected val notifications = Notifications(List(s"/notifications/$notificationId1", s"/notifications/$notificationId2"))
 
-      val validRequest = FakeRequest("GET", "/").withHeaders(validHeaders: _*)
+      val validRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("GET", "/").withHeaders(validHeaders: _*)
     }
 
     "return all notifications" in new SetupGetAllNotifications {
+      val expectedXml = <resource href="/notifications/"></resource>
+
       when(mockApiNotificationQueueService.getNotifications()(any(classOf[HeaderCarrier])))
         .thenReturn(Future.successful(notifications))
 
+      when(mockXmlBuilder.toXml(notifications)).thenReturn(expectedXml)
+
       val result = await(controller.getAll().apply(validRequest))
 
       status(result) shouldBe OK
 
-      val expectedXml = scala.xml.Utility.trim(
-        <resource href="/notifications/">
-          <link rel="self" href="/notifications/"/>
-          <link rel="notification" href="/notifications/1234"/>
-          <link rel="notification" href="/notifications/6789"/>
-        </resource>
-      )
-
       string2xml(bodyOf(result)) shouldBe expectedXml
```

Comment:
I am confues. `notifications` contains 2 elements, but `expectedXml` singular - is this expected?